code/WorkInProgress/TempEngine.dm

```diff
@@ -154,8 +153,12 @@
 			equalize_gases(list(gas_input,gas_output))
 
 		if(is_circulator_active())
-			if(!src.lubed && prob(5))
-				src.audible_message("<span class='alert'>[src] makes an unsettling grinding sound!</span>")
+			if(prob(5))
+				switch(src.lube_boost)
+					if(0.0 to 0.8)
+						src.audible_message("<span class='alert'>[src] makes an unsettling grinding sound!</span>")
+					if(0.81 to 1.0)
```

Comment:
you'll want to change this to be 0.8 to 1.0, otherwise you'll miss all the floating point values, the first would get processed first, i'd say probably return?

Reply:
Thanks good catch.
Bad copypasta, (0.8, 0.81) would be ignored per DM reference ☹️